meteor-app/imports/api/parts/methods.js

```diff
@@ -0,0 +1,25 @@
+import { Meteor } from 'meteor/meteor'
+import Parts from '/imports/parts/parts'
+import log from '/imports/lib/server/log'
+
+const debug = require('debug')('att:server-methods')
+
+Meteor.methods({
+    'parts.insert'(part) {
+      try {
+        return Parts.insert(part)
+      } catch (e) {
+        log.error({ e })
+        throw new Meteor.Error(500, e.sanitizedError.reason)
+      }
+    },
+    // 'parts.update'(id, wholesalePrice) {
```

Comment:
I'm not keen on this (commented out blocks of code) - please be bold and put code in place, as you have gone to the effort of coding it, or just delete it.

Git is very good at tracking changes, so I would prefer not to find blocks of code that are commented out